lightning/src/events/mod.rs

```diff
@@ -480,12 +480,16 @@ pub enum HTLCHandlingType {
 		channel_id: ChannelId,
 	},
 	/// Scenario where we are unsure of the next node to forward the HTLC to.
+	///
+	/// Deprecated: will only be used in versions before LDK v0.2.0.
```

Comment:
I still see one occurrence of HTLCHandlingType::UnknownNextHop in a macro in `process_pending_htlc_forwards`

Reply:
Thanks!